src/components/menu/__internal__/submenu/submenu.spec.js

```diff
@@ -748,4 +749,28 @@ describe("Submenu component", () => {
       );
     });
   });
+
```

Comment:
Would be good to add a test for keyboard nav with a scrollable block just so we can catch it if there's a regression

Reply:
I did think about it but the keyboard nav just loops over `MenuItems` - so the test below proving that the scrollable block renders all of it's child `MenuItems` is enough I think, otherwise we're just testing the keyboard nav functionality twice

Reply:
Yeah that's a fair point, as long as any regression in that behaviour is caught then all good